Parallel Computing with Dask.ipynb

```diff
@@ -1781,10 +1791,10 @@
     "So\n",
     "\n",
     "```\n",
-    "[1974, 1975, 1976]\n",
+    "[1974, 1975, 1975]\n",
```

Comment:
FWIW, I somewhat prefer the first one, because I think it's the natural way people approach this. It took me a second to get my head around this. The docs may be clearer now. The intention here was to show a "gotcha."